.github/workflows/build-dev-image.yml

```
@@ -0,0 +1,32 @@
+on:
+  push:
+    paths:
+      - 'Dockerfile.dev'
+      - 'package.json'
+      - 'pnpm-lock.yaml'
+  workflow_dispatch:
+
+jobs:
+  build:
+    # Skip the whole job if the key files aren’t present
+    if: ${{ hashFiles('Dockerfile.dev') != '' && hashFiles('package.json') != '' }}
```

actionlint (1.7.7)

calling function "hashFiles" is not allowed here. "hashFiles" is only available in "jobs.<job_id>.steps.continue-on-error", "jobs.<job_id>.steps.env", "jobs.<job_id>.steps.if", "jobs.<job_id>.steps.name", "jobs.<job_id>.steps.run", "jobs.<job_id>.steps.timeout-minutes", "jobs.<job_id>.steps.with", "jobs.<job_id>.steps.working-directory". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)